packages/renderer/src/lib/image/ImagesList.spec.ts

```diff
@@ -223,7 +206,7 @@ test('Expect two images in list given image id and engine id', async () => {
       Status: 'Running',
       engineId: 'podman',
       engineName: 'podman',
-    },
+    } as unknown as ImageInfo,
```

Comment:
suggestion (non-blocking): You may be able to cast the full array like `as unknown as ImageInfo[]` to avoid casting individual item